util/util.go

```diff
@@ -382,3 +384,52 @@ func ConfigEnvWithCredential(backupTarget string, credentialSecret string, conta
 	}
 	return nil
 }
+
+func GetInitiatorNSPath() string {
+	initiatorNSPath := "/host/proc/1/ns"
+	pf := iscsi_util.NewProcessFinder("/host/proc")
+	ps, err := pf.FindAncestorByName("dockerd")
+	if err != nil {
+		logrus.Warnf("Failed to find dockerd in the process ancestors, fall back to use pid 1: %v", err)
+	} else {
+		initiatorNSPath = fmt.Sprintf("/host/proc/%d/ns", ps.Pid)
+	}
+	return initiatorNSPath
+}
+
+func GetDiskCapacity(directory string) (map[string]interface{}, error) {
```

Comment:
Call it `GetDiskInfo` instead, since you will get `fsid` as well.

Also, create an object for returning. No need to do type assertion.